package.json

```diff
@@ -40,25 +36,13 @@
     "css-loader": "^3.5.3",
     "dedent": "^0.7.0",
     "del-cli": "^3.0.1",
-    "design-system": "github:ProtonMail/design-system.git#semver:^1.5.6",
-    "ejs-webpack-loader": "^2.2.2",
-    "eslint": "^7.7.0",
-    "eslint-config-airbnb-base": "^14.1.0",
-    "eslint-config-react-app": "^5.2.1",
-    "eslint-plugin-flowtype": "^5.1.3",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.1.0",
     "execa": "^4.0.2",
     "fast-sass-loader": "^1.5.0",
     "favicons-webpack-plugin": "^3.0.1",
     "file-loader": "^6.0.0",
-    "file-saver": "^2.0.2",
```

Comment:
we don't need it anymore ?

Reply:
yes we need it but it's depended on from the project that uses it instead

Reply:
what do you mean?

Reply:
it's not a webpack loader/plugin :facepalm: